### ISSUE TITLE
UserWarning of `scheduler.step()` and warning of Grad strides do not match bucket view strides

### ISSUE THREAD

**Issue by HWPengSYSU1993** [hitachinsk/FGT#21]
**Is it normal to get the following warnings when I use multiple GPUs to run the code? And when I run the code with a single GPU, both of the following warnings are missing**

_UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible.
Warning: Grad strides do not match bucket view strides._

![image](https://user-images.githubusercontent.com/29362830/205919630-4bafb7f4-88b9-43a2-9606-58fffad8ee83.png)

**Comment by hitachinsk** [comment]
I also encountered these warnings, but currently I don't know how to avoid them. I just ignored them.
